mobile/src/modules/price-api/store/reducer.ts

```diff
@@ -52,6 +63,19 @@ export interface HistoricalPriceInfo {
   HasWarning?: boolean;
 }
 
+export enum PriceType {
+  BURST,
+  BTC,
+  USD
+}
+export type PriceTypeStrings = keyof typeof PriceType;
```

Comment:
and after comment above you can use it like Array<PriceType> without this type at all